src/Process/Runner/TestRunConstraintChecker.php

```diff
@@ -44,10 +44,16 @@ final class TestRunConstraintChecker
 {
     public const MSI_FAILURE = 'min-msi';
 
+    /**
+     * @deprecated
+     */
     public const COVERED_MSI_FAILURE = 'min-covered-msi';
```

Comment:
Just dropping a use case here in case you're not aware of it. We require that all mutations are prevented from escaping. We use `min-covered-msi=100` so that if code isn't covered at all then the build can still be green.

We then report back the code coverage stats in pull requests so that reviewers can manually decide if the code coverage is high enough (we expect 100% but allow some exceptions in certain circumstances).

I _think_ if the `min-covered-msi` option was removed then we'd have to remove the 100% limit, and therefor rely on reviews to manually check if any mutants had escaped, and then in cases where coverage wasn't 100% to figure out whether it's acceptable they escaped or not, this would be much worse than our process today.

Reply:
That ability should not be removed don't worry! It may however change a bit (e.g. a change of option name) depending on the decision taken in #1064